frontend/src/pages/offers/LineItemForm.tsx

```diff
@@ -117,6 +138,7 @@ const LineItemForm: FunctionComponent<Props> = ({
           minLength={5}
           register={register}
           errors={errors}
+          helpText="Pallets with comprehensive descriptions are more likely to get picked up."
```

Comment:
We discussed this on Sunday: we want to encourage people to write good descriptions for each line item.